app/global/db/Bluementa.sql

```diff
@@ -0,0 +1,184 @@
+-- phpMyAdmin SQL Dump
+-- version 4.9.5deb2
+-- https://www.phpmyadmin.net/
+--
+-- Host: localhost:3306
+-- Generation Time: May 09, 2023 at 01:49 PM
+-- Server version: 8.0.32-0ubuntu0.20.04.2
+-- PHP Version: 7.4.3-4ubuntu2.18
+
+SET SQL_MODE = "NO_AUTO_VALUE_ON_ZERO";
+SET AUTOCOMMIT = 0;
+START TRANSACTION;
+SET time_zone = "+00:00";
+
+
+/*!40101 SET @OLD_CHARACTER_SET_CLIENT=@@CHARACTER_SET_CLIENT */;
+/*!40101 SET @OLD_CHARACTER_SET_RESULTS=@@CHARACTER_SET_RESULTS */;
+/*!40101 SET @OLD_COLLATION_CONNECTION=@@COLLATION_CONNECTION */;
+/*!40101 SET NAMES utf8mb4 */;
+
+--
+-- Database: `Bluementa`
+--
+
+CREATE DATABASE Bluementa;
```

Comment:
```suggestion
CREATE DATABASE Bluementa;
```
Nem jo az adatbazis nev